ChangeLog.d/clean_pem_buffers.txt

```diff
@@ -0,0 +1,13 @@
+Bugfix
+  * As an optimisation to stop large buffers being written to the stack the
+    code was changed to write the raw der data to the same buffer that the
+    pem data would later be written to. Unfortunately even though the pem data
+    was always NULL terminated the der data would be left in the buffer after
```

Comment:
“NULL” is only capitalized when it's the C constant (including when the name of the constant is used metonymically to mean a null pointer). Here you mean a null byte, so null is an ordinary adjective, written in lowercase.